packages/core/app-layout/src/components/AppLayout.vue

```diff
@@ -11,6 +11,7 @@
     </div>
     <AppNavbar
       v-if="!navbar.hidden"
+      :key="String(sidebar.hidden)"
```

Comment:
This triggers the `AppNavbar` component to remount (re-render) if the sidebar is toggled.